src/platform/nrfconnect/BUILD.gn

```diff
@@ -75,6 +75,20 @@ static_library("nrfconnect") {
     }
   }
 
+  if (chip_enable_wifi) {
+    sources += [
+      "../Zephyr/WiFiManager.h",
+      "ConnectivityManagerImplWiFi.cpp",
+      "ConnectivityManagerImplWiFi.h",
+      "NetworkCommissioningDriver.cpp",
+      "NetworkCommissioningDriver.h",
+    ]
+    if (chip_mdns == "platform") {
+      sources += [ "../Zephyr/DnssdImpl.cpp" ]
```

Comment:
Can we at least move the DnssdImpl and WiFiManager to separate WiFi directory? Now it looks like DnssdImpl should be used for every Zephyr platform use case, what might be misleading.

Reply:
When it comes to DnssdImpl I think we will end up using minimal mdns, anyway, so it's not worth moving it anywhere for now.

Reply:
I will keep it for now since, as Damian said, the DNS will be affected anyway and it makes not much sense to move only one file (mainly WiFiManager) to separate directory.